### PR TITLE
[SEE-DESCR] LPS-53819 Autogenerated

### PR DESCRIPTION
**Pull request by adolfopa**
Hi @sergiogonzalez 

Once this PR gets merged I'll send another one taking care of the new exception in some of the plugins.

Thanks!

### CODE REVIEW

**Comment by adolfopa** [comment]
ci:retest

File: portal-impl/src/com/liferay/portal/verify/VerifyDocumentLibrary.java [hunk 1]
```diff
@@ -369,12 +370,9 @@ public void performAction(Object object) {
 							dlFileEntry.getFileEntryId(),
 							dlFileEntry.getFileName(), dlFileEntry.getTitle());
 					}
-					catch (PortalException pe) {
-						if (!(pe instanceof DuplicateFileException) &&
-							!(pe instanceof DuplicateFolderNameException)) {
-
-							return;
-						}
+					catch (DuplicateFileException |
```
Comment (jorgediaz-lr) [inline portal-impl/src/com/liferay/portal/verify/VerifyDocumentLibrary.java]: @adolfopa This DuplicateFileException can be removed, we only want to rename title when DuplicateFileEntryException is thrown inside validateFile

File: portal-impl/src/com/liferay/portal/verify/VerifyDocumentLibrary.java [hunk 2]
```diff
@@ -547,12 +547,8 @@ protected void renameDuplicateTitle(DLFileEntry dlFileEntry)
 
 				return;
 			}
-			catch (PortalException pe) {
-				if (!(pe instanceof DuplicateFolderNameException) &&
-					 !(pe instanceof DuplicateFileException)) {
-
-					throw pe;
-				}
+			catch (DuplicateFileException | DuplicateFileEntryException |
```
Comment (jorgediaz-lr) [inline portal-impl/src/com/liferay/portal/verify/VerifyDocumentLibrary.java]: @adolfopa This DuplicateFileException can be removed, we only want to rename title when DuplicateFileEntryException is thrown inside validateFile

File: portal-impl/src/com/liferay/portlet/documentlibrary/lar/FileEntryStagedModelDataHandler.java [hunk 3]
```diff
@@ -537,7 +538,7 @@ protected void doImportStagedModel(
 					fileEntry.getTitle(), fileEntry.getDescription(), null, is,
 					fileEntry.getSize(), serviceContext);
 			}
-			catch (DuplicateFileException dfe) {
+			catch (DuplicateFileException | DuplicateFileEntryException dfe) {
```
Comment (jorgediaz-lr) [inline portal-impl/src/com/liferay/portlet/documentlibrary/lar/FileEntryStagedModelDataHandler.java]: @adolfopa This DuplicateFileException can be removed, we only want to rename title when DuplicateFileEntryException is thrown inside addFileEntry, if a DuplicateFileException is received, a title rename of fileEntry will not help to fix the problem during **DLAppLocalServiceUtil.addFileEntry**

**Comment by sergiogonzalez** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by sergiogonzalez** [comment]
Adolfo will resend. We won't handle DuplicateFileException in the UI (because the user cannot do anything to fix it) and in the future we will implement a Wrapper in the store so if it's invoked from DL and it throws a DuplicateFileException (and the file doesn't exist in the database) it retries the operation.

**Pull request closed without merging**